Modules/Sources/Networking/Remote/RefundsRemote.swift

```diff
@@ -145,3 +145,27 @@ public extension RefundsRemote {
         static let include: String    = "include"
     }
 }
+
+extension RefundsRemote: POSRefundsRemoteProtocol {
+    public func loadRefunds(for siteID: Int64, by orderID: Int64, with refundIDs: [Int64]) async throws -> [Refund] {
+        return try await withCheckedThrowingContinuation { continuation in
+            loadRefunds(for: siteID, by: orderID, with: refundIDs) { refunds, error in
+                if let error = error {
+                    continuation.resume(throwing: error)
+                } else {
+                    continuation.resume(returning: refunds ?? [])
```

Comment:
Do you know why does the type returned from the remote is `[Refund]?, Error?`? 

We should either have an array of refunds (which ofc could be empty), or an error. I'm not sure what means for these to be `nil`, and the API reference doesn't say anything about it. 

Out of the scope of this PR since you've only made an async wrapper of what was already there, but worth noting it down for investigating further.